README.md

```diff
@@ -37,7 +37,8 @@ Params:
 | ---------- | ----------- | ---- | ------------- |
 | contentBlockPath | Directory of the block | string | Current dir |
 | entryPath | Path to the entry file exporting `block` and `settings` | string | `src/index.tsx` |
-| port | Port for the local web server | string | 5600 |
+| port | Port for the local development server | number | 5600 |
+| metaPort | Port for the local meta information (url, entrypoint, port) | number | 5601 |
```

Comment:
You can probably register the meta endpoint on the vite server

Reply:
using middleware mode now: https://github.com/Frontify/frontify-cli/pull/232/commits/a67394644891ab94952be5f58b807d7ce16c4d3c